alpha/lib/model/LiveEntry.php

```diff
@@ -474,6 +475,28 @@ public function canViewExplicitLive()
 		return true;
 	}
 
+	/**
+	 * @param int $time
+	 * @return array<LiveStreamScheduleEventable>
+	 */
+	public function getScheduleEvents($time = null)
+	{
+		$events = array();
+		$pluginInstances = KalturaPluginManager::getPluginInstances('IKalturaScheduleEventGetter');
+		foreach ($pluginInstances as $getEventer)
+		{
+			/* @var $getEventer IKalturaScheduleEventGetter */
+			$pluginEvents = $getEventer->getScheduleEvents($this->getId(), array(ScheduleEventType::LIVE_STREAM), $time);
+			if (count($pluginEvents))
```

Comment:
count will not work on null, and if its empty array it will be false anyway - please remove the count.